### ISSUE TITLE
Y24-148 - As a team lead (Danni) I would like to be able to scan the location when I import samples on the reception page. This would speed up the process as we would not also need to visit labwhere

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#1779]
**User story**
As a team lead (Danni) I would like to be able to scan the location when I import samples on the reception page. This would speed up the process as we would not also need to visit labwhere

**Who are the primary contacts for this story**
Steve / Danni / James

**Who is the nominated tester for UAT**
Steve / Danni / James

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] When I visit the general reception page I want to be able to select a workflow (optional)
- [x] When that workflow is selected I need to scan my swipe card id
- [x] Each workflow will correspond to a location in labwhere
- [x] When the workflow is selected it will show on the page which location the samples will be scanned inot
- [x] When I create the samples in the reception the labware for each sample will be scanned into labwhere
- [x] This location will then appear when you visit the samples page
- [x] The workflows will exist as configuration in the ui with each workflow having a name a location name and a location barcode
- [x] The user can only select 1 workflow for each set of samples being imported

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #714

**Additional context**
- Danni will provide a list of workflows and their corresponding location barcodes
- The workflow table in the service will need to be changed to add the location_barcode (didn't see the need for a separate story).

**Comment by stevieing** [comment]
Danni will provide a list of workflows along with their locations.
We can add that to the service.

**Comment by stevieing** [comment]
Workflows & locations:
Pacbio -20 samples (LRT006 Draw 1):        lw-shelf-1-30472
Pacbio Fridge samples (LRT007 – Shelf 1):    lw-shelf-1-30451
ONT -20 samples (LRT020 Draw 1):          lw-drawer-1-37292
ONT Fridge samples (LTR018 Shelf 1):       lw-shelf-1-30503
Extractions -80 samples (Long Read DTOL Freezer 2/Shelf 3/Rack 3) : lw-drawer-2-30398

We need to decide how to represent these in the ui or service. Might be best to hardcode in the ui side for now.
